config/test.js

```diff
@@ -2,4 +2,5 @@
 module.exports = {
   apiHost: 'https://localhost',
   allowErrorSimulation: true,
+  enableClientConsole: true,
```

Comment:
Will this make test output (any) noiser? I imagine the aim of setting this to `false` here was to make tests not output a log of logging stuff.